crc32.c

```diff
@@ -20,13 +20,12 @@
 
   DYNAMIC_CRC_TABLE and MAKECRCH can be #defined to write out crc32.h.
  */
-#ifdef HAS_PCLMUL
 
-#include <emmintrin.h>
-#include <smmintrin.h>
-#include <wmmintrin.h>
-//#include <stdio.h>
-#include <cpuid.h>
+#ifdef HAS_PCLMUL
+ #include <emmintrin.h>
+ #include <smmintrin.h>
+ #include <wmmintrin.h>
```

Comment:
I think all of those can be replaced by a single `#include <immintrin.h>`

Reply:
defines no longer in crc32, @Adenilson can decide if these can be replaced in arc_simd.c